scripts/buf-breaking.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+AGAINST=${1:-buf.build/chain4travel/camino-messenger-protocol}
+EXCLUDE=""
+
+if [ -f "missing_files.txt" ] ; then
+	EXCLUDE="--exclude-path $(cat missing_files.txt | sed -e's#proto/##g' | tr "\n" "," | head -c -1)"
+fi
+
+buf breaking $EXCLUDE --against "$AGAINST"
```

Comment:
Should we add some echo lines so we have some output in the action log?

```suggestion
#!/bin/bash

echo "Starting buf breaking check..."

AGAINST=${1:-buf.build/chain4travel/camino-messenger-protocol}
EXCLUDE=""

echo "AGAINST: ${AGAINST}"

if [ -f "missing_files.txt" ] ; then
        echo "missing_files.txt files exists in current dir, building exclude list..."
        EXCLUDE="--exclude-path $(cat missing_files.txt | sed -e's#proto/##g' | tr "\n" "," | head -c -1)"
fi

echo "EXCLUDE: '${EXCLUDE}'"

echo "Running buf breaking..."
buf breaking $EXCLUDE --against "$AGAINST"
```

Reply:
✔️ Added better logging in the script